tests/specs/config_changes/RemovingOakThenDax.txt

```diff
@@ -161,24 +161,20 @@
 {
   "version": "4",
   "specifiers": {
-    "jsr:@dsherret/dax": "jsr:@dsherret/dax@0.0.1",
-    "jsr:@dsherret/which@0.0.1": "jsr:@dsherret/which@0.0.1",
-    "jsr:@std/assert@0.210": "jsr:@std/assert@0.210.0",
-    "jsr:@std/assert@^0.210.0": "jsr:@std/assert@0.210.0",
```

Comment:
You'll notice here that we got smarter and can tell these are actually the same version requirement now.